nuxeo-wopi-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-wopi/nuxeo-wopi-link.html

```diff
@@ -28,12 +28,10 @@
       }
     </style>
 
-    <template is="dom-if" if="[[_isAvailable(blob)]]">
```

Comment:
I may keep this `dom-if` because even if we want to let people change availability with the `nuxeo-filter` in slot contrib. It is pointless to display the link if url is empty, nope? (I know, I said the opposite before)